File: HexAssetUI/src/app/models/AuditRequest.ts
export class AuditRequestModel {
    auditId: number;
    userId: 0;
    allocationId: 0;
    item: string;
    auditStatus: string;
    auditDate: Date

    constructor() {
        this.auditId = 0;
        this.userId = 0;
        this.allocationId=0;
        this.item = "";
        this.auditStatus = "";
        this.auditDate = new Date();
    }
}

File: HexAssetAPI/Models/AuditRequest.cs
using System.ComponentModel.DataAnnotations;

namespace HexAsset.Models
{
	public class AuditRequest
	{
		[Key]
		public int AuditId { get; set; }
		public int UserId { get; set; }

		public int? AllocationId { get; set; }
		public required string AuditStatus { get; set; }

		public string? Item { get; set; }
		public DateTime AuditDate { get; set; } = DateTime.UtcNow;

		public User? User { get; set; }
	}
}
